spec/unit/octo_fetcher_spec.rb

```diff
@@ -265,7 +265,7 @@
                             "diff_url"  => "https://github.com/skywinder/changelog_test/pull/21.diff",
                             "patch_url" => "https://github.com/skywinder/changelog_test/pull/21.patch" },
                         "body"           =>
-                          "to test https://github.com/skywinder/github-changelog-generator/pull/305\r\nshould appear in change log with #20" }
+                          "to test https://github.com/skywinder/github-changelog-generator/pull/305\r\nshould appear in changelog with #20" }
```

Comment:
Undo the changes in this file.

They refer to data in another repository, the changing of which is outside the scope of this find-and-replace PR.